taxi/models.py

```diff
@@ -0,0 +1,23 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    def __str__(self: 'Manufacturer') -> str:
+        return f"{self.name}, {self.country}"
```

Comment:
Avoid overusing f-strings when they are not necessary. You can simply return `self.name + ', ' + self.country` instead.